CHANGES.rst

```diff
@@ -2,10 +2,11 @@
 History
 =======
 
-2.1 (unreleased)
+2.0.1 (unreleased)
```

Comment:
As you are dropping support for Python 3.5 the next version should not be a bugfix version.
Additionally the drop of Python 3.5 and additional support of 3.7 should be mentioned here.

Reply:
Hm, should we keep 3.5 for now then and just add 3.7 to the mix?

Reply:
I readded 3.5 so we can keep the bugfix release version, but i added 3.7.